packages/fxa-event-broker/test/lib/api/proxy-controller.spec.ts

```diff
@@ -0,0 +1,169 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as hapi from '@hapi/hapi';
+import { assert as cassert } from 'chai';
+import * as jwtool from 'fxa-jwtool';
+import 'mocha';
+import { Logger } from 'mozlog';
+import * as nock from 'nock';
+import { assert, SinonSpy } from 'sinon';
+import * as sinon from 'sinon';
+import { stubInterface } from 'ts-sinon';
+
+import Config from '../../../config';
+import { SUBSCRIPTION_STATE_EVENT_ID } from '../../../lib/jwts';
+import * as proxyserver from '../../../lib/proxy-server';
+import { ClientWebhookService } from '../../../lib/selfUpdatingService/clientWebhookService';
+
+const TEST_KEY = {
```

Comment:
I'm also fuzzy on JWT keys & signing, but the tests pass and they look clever so :+1: